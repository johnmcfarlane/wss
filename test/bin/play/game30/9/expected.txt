 50  2,6  | DUPLA
 42  3,15 - TAPUL
 42 12,1  | UPHELD
 39 12,1  | UPHEAL
 36 12,1  | UPLEAD
 33 12,1  | PULED
 30 12,1  | PALED
 29  2,10 | APH
 29  7,11 - LAP
 27 12,1  | DUPE
 27 12,1  | HALED
 27  7,11 - LAH
 27  7,11 - LAUD
 27  7,11 - LHD
 27 12,1  | LUPE
 27  1,15 - PATH
 27 12,1  | PULE
 26  2,7  | HUPA
 26  7,11 - LUAU
 26  7,11 - LUD
 25  9,3  - APH
 25  7,11 - LAD
 25  9,3  - PAH
 24 10,1  | APHIDS
 24  2,14 - DIALUP
 24 12,1  | PALE
 24  1,15 - PATD
 24  1,15 - PATU
 24  3,15 - TAPU
 24  3,15 - THUD
 24  3,15 - TPH
 24  2,7  | UPLA
 23 10,3  - LHD
 22  3,3  - DUPLA
 22  2,12 | HAPU
 22  2,7  | HULA
 22  8,5  - LHD
 22  7,11 - LU
 22 10,1  | PLAIDS
 21 11,3  - APH
 21  1,15 - AUTH
 21  9,3  - DAH
 21 10,3  - DHA
 21 12,1  | HADE
 21 12,1  | HALE
 21 10,3  - HAP
 21  2,12 | HHD
 21  7,11 - LA
 21  9,3  - LAH
 21  1,15 - LATH
 21 10,3  - PAH
 21  1,7  - PULED
 21  3,15 - THAD
 21  3,15 - TULU
 21  3,15 - TUP
 21  2,15 - UTAH
 20  2,11 | DHAL
 20 12,4  | EUPAD
 20 10,1  | HALIDS
 20  6,1  | HAPU
 20 12,3  - HAPU
 20  1,7  - PALED
 20 12,3  - PHU
 20  2,14 - PILAU
 20  5,3  - PLU
 20  2,8  | PUA
 20  6,1  | PUDU
 20  6,1  | PULU
 20  4,3  - PULU
 20  2,7  | ULUA
 20  4,3  - UPLA
 19 10,3  - DAP
 19 10,8  | DUPLA
 19  4,3  - HULU
 19 10,3  - LAP
 19 10,3  - PAD
 19 13,2  | PURDAH
 19 13,2  | PURUHA
 18  4,14 | DA
 18 12,1  | DALE
 18  2,8  | DHA
 18 10,1  | DULIAS
 18 12,3  - HALP
 18 12,3  - HAP
 18 10,1  | HAPI
 18  2,14 - HILDA
 18  6,1  | HULU
 18 12,1  | LADE
 18 12,3  - PAH
 18 10,1  | PAHI
 18 10,2  | PAILS
 18 12,3  - PAUL
 18  6,2  | PHU
 18  1,9  | PHUT
 18  2,15 - PTA
 18 10,1  | PULI
 18  3,15 - TALD
 18  3,15 - TAP
 18  3,15 - TULA
 18  1,15 - ULTA
 18 10,7  - UPLA
 17  3,6  | APH
 17  4,3  - AULU
 17 10,3  - DAH
 17  9,3  - DHA
 17  3,3  | DUPLA
 17 10,3  - HAD
 17  1,7  - HALED
 17  2,12 | HAP
 17  4,3  - HULA
 17  3,5  | HUP
 17 10,3  - LAH
 17  9,1  | PAHA
 17  1,9  | PHAT
 17  7,1  | PLAN
 17  7,1  | PUAN
 17  1,7  - PULE
 17 10,6  | SADDHU
 16 10,3  - AH
 16 10,7  - APH
 16 12,3  - DAP
 16 12,3  - DHAL
 16 12,3  - DHU
 16  9,2  | DUALI
 16  1,8  | DULUTH
 16 10,8  | DUP
 16 11,3  - HA
 16 10,2  | HAILS
 16  3,4  | HALP
 16  3,5  | HAP
 16 13,2  | HARP
 16 12,3  - HAUL
 16  9,1  | HUPA
 16  7,8  | JOEL
 16  2,9  | PA
 16 12,3  - PAD
 16  3,5  | PAH
 16 12,3  - PAL
 16  9,1  | PALA
 16  1,7  - PALE
 16 10,1  | PALI
 16  9,3  | PALI
 16 12,3  - PAU
 16  9,1  | PAUA
 16  1,9  | PAUT
 16 10,2  | PHI
 16  1,9  | PLAT
 16  6,2  | PLU
 16 13,2  | PURDA
 16  2,8  | ULA
 16  5,3  - ULU
 16  5,12 | WHUP
 15  3,6  | ADP
 15 10,9  - APH
 15 10,8  | DAP
 15  2,11 | DHA
 15  3,5  | DHAL
 15  3,5  | DUP
 15  2,12 | HAH
 15  3,6  | HAP
 15  3,3  | HAULD
 15 13,3  - HUP
 15 10,3  - LAD
 15  3,3  - PAUL
 15  3,3  | PLAUD
 15  3,4  | PLUD
 15  3,5  | PUD
 15 13,2  | PURAU
 15 13,2  | PURL
 15 13,2  | PURU
 15  3,15 - THA
 15 10,7  - UP
 15  3,4  | UPLA
 15  2,15 - UTU
 15  5,12 | WHAP
 14 12,3  - AHU
 14 10,2  | ALIDS
 14 13,3  - APH
 14  6,1  | AULU
 14 12,3  - DAH
 14 10,1  | DAHI
 14  9,3  | DALI
 14  3,5  | DAP
 14  3,6  | DAP
 14 12,3  - DHA
 14 10,1  | DHAI
 14  6,2  | DHU
 14 13,3  | DRUPAL
 14  3,5  | DUAL
 14  2,9  | HA
 14 12,3  - HAD
 14 12,3  - HAL
 14 12,3  - HAU
 14  3,3  - HAUL
 14  9,1  | HULA
 14  3,4  | HUPA
 14 10,2  | LAIUS
 14  3,5  | LAP
 14  3,6  | LAP
 14  3,6  | LHD
 14  6,1  | LUAU
 14 10,3  - PA
 14  3,5  | PAD
 14  3,6  | PAH
 14 13,2  | PARD
 14  6,2  | PAU
 14  4,3  - PDL
 14  3,6  | PDL
 14 10,2  | PLI
 14  4,3  - PUL
 14  3,5  | UDAL
 14 10,7  - UDAL
 14  5,3  - ULA
 14 10,7  - ULUA
 14  5,12 | WHUD
 13  3,5  | ADP
 13 10,7  - ADP
 13 10,7  - AHU
 13  3,5  | ALP
 13 10,7  - ALP
 13  3,5  | APH
 13 10,7  - AULD
 13 10,7  - AULU
 13  3,3  - DHAL
 13  1,8  | DUHAT
 13  1,7  - DUPE
 13  2,12 | HAD
 13  1,7  - HADE
 13  2,12 | HAL
 13  9,1  | HALA
 13  1,7  - HALE
 13  1,9  | HALT
 13 13,3  - HAP
 13  1,9  | HAUT
 13 13,3  - HLD
 13  3,5  | HLD
 13 13,3  - HUD
 13  3,5  | HUD
 13 13,2  | HURL
 13  1,7  - LUPE
 13  3,5  | PUA
 13 10,6  | SADHU
 13 10,7  - UH
 13 10,7  - ULU
 13  9,1  | UPLA
 13  5,12 | WHAU
 12 10,9  - ADP
 12  1,8  | ADULT
 12 11,3  - AH
 12  6,2  | AHU
 12 10,9  - AHU
 12 10,9  - ALP
 12 10,7  - AUH
 12  3,4  | AULD
 12 10,9  - AULD
 12 10,9  - AULU
 12  2,9  | DA
 12  3,5  | DAH
 12  3,6  | DAH
 12 10,8  | DAH
 12 12,3  - DAL
 12 10,1  | DALI
 12 12,3  - DAU
 12 12,3  | DELPH
 12  7,1  | DHAN
 12 13,3  | DRUPA
 12  3,3  - DUAL
 12  1,8  | DULAT
 12 10,3  - HA
 12  3,5  | HAD
 12 13,2  | HARD
 12 13,2  | HARL
 12  1,10 - HATI
 12  6,2  | HAU
 12  3,7  - HEP
 12 10,2  | HUI
 12  3,4  | HULA
 12  2,9  | LA
 12  3,5  | LAH
 12  3,6  | LAH
 12  1,9  | LAUTU
 12  3,5  | LHD
 12  6,3  - LU
 12  9,3  - PA
 12 12,3  - PA
 12  2,14 | PA
 12  3,6  | PAD
 12  4,3  - PAL
 12  3,6  | PAL
 12 12,3  | PEUHL
 12 13,1  | PHAR
 12 13,2  | PHR
 12 13,3  | PRAHU
 12 13,3  | PRUH
 12 13,4  | RALPH
 12  3,15 - TAD
 12  3,15 - TAL
 12  3,15 - TAU
 12  3,15 - TH
 12  3,15 - TUA
 12  3,3  - UDAL
 12 10,7  - ULA
 12  6,2  | ULU
 12  7,3  | UNAU
 12  2,15 - UTA
 12  5,12 | WAUP
 12  5,12 | WUDU
 12  5,12 | WUP
 11 13,3  - ADP
 11 10,7  - AH
 11 13,3  - AHU
 11 10,7  - ALD
 11 12,2  | ALEPH
 11 13,3  - ALP
 11 10,7  - AUD
 11  3,5  | AUH
 11 10,9  - AUH
 11  4,3  - AUL
 11 10,7  - AUL
 11 10,8  | DADU
 11 10,8  | DAUD
 11  7,1  | DAUN
 11  3,7  - DEP
 11 10,8  | DHU
 11  7,1  | DUAN
 11 11,1  | DUHR
 11  1,9  | DULT
 11 13,2  | DURAL
 11 13,3  - HAD
 11  3,6  | HAD
 11  4,3  - HAL
 11 13,3  - HAL
 11  3,6  | HAL
 11 13,3  - HAU
 11  1,8  | HAULT
 11 13,2  | HURA
 11  3,4  | LAUD
 11  7,1  | LAUN
 11  3,7  - LEP
 11  3,5  | LUD
 11  3,7  - PEH
 11 12,2  | PHEAL
 11  1,14 - PHIAL
 11 11,2  | PHR
 11  2,7  - PLED
 11 11,1  | PLUR
 11 13,1  | PLUR
 11 13,2  | PUR
 11 11,2  | PURL
 11 11,2  | PURU
 11  7,1  | ULAN
 11  9,1  | ULUA
 11  3,4  | ULUA
 11  3,6  | UP
 11  5,12 | WHA
 10 11,3  - AD
 10  9,3  - AH
 10 12,3  - AH
 10  3,7  | AH
 10 10,9  - AH
 10 10,2  | AHI
 10 10,9  - ALD
 10  2,7  - APED
 10 11,2  | APR
 10 10,9  - AUD
 10 13,3  - AUH
 10 10,9  - AUL
 10 10,3  - DA
 10  4,3  - DAL
 10  3,6  | DAL
 10  1,7  - DALE
 10  1,9  | DALT
 10  6,2  | DAU
 10  1,8  | DAULT
 10  1,9  | DAUT
 10  3,5  | DHA
 10 10,8  | DHAL
 10 10,8  | DUD
 10 13,1  | DUHR
 10 10,2  | DUI
 10  9,3  - HA
 10 12,3  - HA
 10  2,14 | HA
 10 11,2  | HARP
 10 12,3  | HELP
 10  2,7  - HUED
 10 11,2  | HURL
 10  6,3  - LA
 10 10,3  - LA
 10  3,5  | LAD
 10  3,6  | LAD
 10  1,7  - LADE
 10 13,2  | LARD
 10  1,10 - LATI
 10  2,14 | LU
 10 10,2  | LUI
 10 13,2  | PAR
 10 12,3  | PEDAL
 10 11,1  | PHAR
 10  1,14 - PHIL
 10 12,2  | PLEAD
 10 13,4  | RAHUL
 10 13,4  | RUPA
 10 10,2  | UDI
 10  5,12 | WAP
 10  5,12 | WAUL
 10  5,12 | WUD
  9 10,7  - AD
  9 13,3  - AH
  9 10,7  - AL
  9 13,3  - ALD
  9  3,5  | ALD
  9 13,3  - AUD
  9  3,5  | AUD
  9 13,3  - AUL
  9 10,8  | DAD
  9 10,8  | DAL
  9 10,8  | DAU
  9  2,14 - DIPL
  9  3,8  - DOA
  9 10,8  | DUAD
  9 10,8  | DUAL
  9  2,7  - DUEL
  9 13,2  | DURA
  9 12,4  | EPHA
  9 13,3  - HA
  9  2,7  - HAED
  9 12,3  | HEALD
  9 12,3  | HEAP
  9 12,3  | HEUAU
  9 10,3  | HIPS
  9 11,2  | HURA
  9 12,3  | LEHUA
  9  3,8  - LOA
  9 13,2  | LURA
  9  1,8  - PAHO
  9 11,2  | PARD
  9  1,10 | PATH
  9  3,7  - PED
  9 12,3  | PELU
  9 12,3  | PEUL
  9  2,7  - PLEA
  9 12,2  | PLED
  9 13,3  | PRAD
  9 13,3  | PRAU
  9 11,2  | PUR
  9 13,4  | RHD
  9  3,6  | UH
  9  3,5  | ULA
  9  1,8  - UPDO
  9 13,3  | URDU
  9  5,12 | WAH
  8  9,3  - AD
  8  3,7  | AD
  8 10,9  - AD
  8  3,6  | AH
  8  5,3  - AL
  8  9,3  - AL
  8  3,7  | AL
  8 10,9  - AL
  8 11,1  | ALUR
  8 13,1  | ALUR
  8 12,2  | APED
  8  2,13 - AR
  8 13,3  | ARDU
  8  9,1  | AULA
  8  9,3  - DA
  8 12,3  - DA
  8  2,14 | DA
  8 11,1  | DAUR
  8 12,3  | DEPA
  8 10,8  | DHA
  8 10,3  | DIPS
  8 11,2  | DURA
  8  2,12 | HA
  8 11,2  | HARD
  8 11,2  | HARL
  8  3,7  - HED
  8  3,7  - HEL
  8 12,3  | HELD
  8 12,3  | HEP
  8  2,14 - HIP
  8 12,2  | HUED
  8  3,14 - IHP
  8  3,14 - IPH
  8  9,3  - LA
  8  2,14 | LA
  8 10,2  | LAI
  8 11,1  | LAUR
  8 12,3  | LEAP
  8 12,3  | LEPA
  8  2,14 - LIPA
  8 10,3  | LIPS
  8 11,2  | LURA
  8  1,14 - PAID
  8  1,14 - PAIL
  8  1,10 | PATD
  8  1,10 | PATU
  8  7,2  | PDN
  8 12,3  | PEAL
  8 12,3  | PEDA
  8 12,3  | PEH
  8  1,14 - PHI
  8  2,8  - PHO
  8  1,10 | PHT
  8  1,15 - PHT
  8  2,14 - PIAL
  8 10,3  | PIUS
  8 12,2  | PLEA
  8  7,2  | PUN
  8 13,4  | RHA
  8  1,12 | TAPU
  8  1,12 | THUD
  8  1,12 | TPH
  8  5,12 | WAD
  7 13,3  - AD
  7 13,3  - AL
  7 13,2  | APR
  7  1,10 | AUTH
  7 13,1  | DAUR
  7  3,7  - DEL
  7 12,3  | DEP
  7 12,3  | DEUL
  7  2,14 - DIP
  7 11,2  | DLR
  7 13,2  | DLR
  7 12,2  | DUEL
  7 11,2  | DUR
  7 13,2  | DUR
  7 12,4  | EDP
  7  4,7  - EH
  7 12,2  | HAED
  7  1,14 - HAIL
  7  1,8  - HALO
  7 12,3  | HEAD
  7 12,3  | HEAL
  7  2,14 - HILA
  7 11,3  | HR
  7  1,14 - HUIA
  7  7,2  | HUN
  7  3,14 - IPL
  7 11,2  | LARD
  7  1,10 | LATH
  7 13,1  | LAUR
  7 12,3  | LEAH
  7  3,7  - LED
  7 12,3  | LEP
  7  3,7  - LEU
  7 12,3  | LEUD
  7  2,14 - LIP
  7  1,8  - LUDO
  7 11,2  | LUR
  7 13,2  | LUR
  7  3,6  | PA
  7  7,2  | PAN
  7 11,2  | PAR
  7  3,7  - PEA
  7 12,3  | PED
  7 10,3  | PIAS
  7  2,14 - PIL
  7  2,14 - PIU
  7  1,14 - PLI
  7  9,2  | PUA
  7  1,10 | PUT
  7  1,15 - PUT
  7 13,4  | RAP
  7 13,4  | RAUL
  7 13,4  | RLD
  7 13,4  | RUD
  7 10,6  | SADH
  7 10,6  | SUDD
  7  1,12 | THAD
  7  1,12 | TULU
  7  1,12 | TUP
  7  2,8  - UPO
  7 13,3  | URAL
  7 11,3  | URD
  7 13,3  | URD
  7 11,3  | URU
  7 13,3  | URU
  7  1,11 | UTAH
  6  3,6  | AD
  6  9,2  | APA
  6 12,2  | APE
  6  2,7  - APE
  6  1,10 | APT
  6  1,15 - APT
  6 13,3  | ARU
  6  2,10 - ATI
  6 13,2  | DAR
  6 12,3  | DEAL
  6  9,2  | DHA
  6  2,14 - DIAL
  6  2,14 - DIAU
  6  7,2  | DUN
  6 12,4  | EDH
  6 12,4  | EPA
  6 12,4  | EUDA
  6  3,6  | HA
  6  7,2  | HAN
  6 12,3  | HED
  6 12,3  | HEL
  6  2,14 - HID
  6 12,2  | HUE
  6  2,7  - HUE
  6  1,14 - HUI
  6  1,10 | HUT
  6  1,15 - HUT
  6 10,4  | IPS
  6  1,14 - LAID
  6 13,2  | LAR
  6 12,3  | LEAD
  6 12,3  | LEDA
  6  2,14 - LIDA
  6 10,3  | LIDS
  6  1,10 | PAT
  6  1,15 - PAT
  6 12,3  | PEA
  6  2,14 - PIA
  6  1,11 | PTA
  6 13,4  | RAH
  6 13,4  | RUA
  6  3,15 - TA
  6  1,12 | TALD
  6  1,12 | TAP
  6  1,12 | TULA
  6  1,10 | ULTA
  6 11,3  | URA
  5  9,2  | AHA
  5  1,14 - AHI
  5  2,8  - AHO
  5  1,10 | AHT
  5  1,15 - AHT
  5 10,3  | AIDS
  5 10,3  | AILS
  5 11,3  | ARU
  5  3,6  | DA
  5  7,2  | DAN
  5 11,2  | DAR
  5  3,7  - DEA
  5 12,3  | DEL
  5 10,3  | DIAS
  5  2,14 - DIL
  5 12,2  | DUE
  5  2,7  - DUE
  5  1,14 - DUI
  5  2,8  - DUO
  5  4,7  - ED
  5  4,7  - EL
  5 12,4  | ELD
  5 12,2  | HAE
  5  2,7  - HAE
  5  2,8  - HAO
  5  1,10 | HAT
  5  1,15 - HAT
  5  2,14 - HIA
  5 10,4  | IHS
  5  3,14 - IUD
  5  3,6  | LA
  5  7,2  | LAN
  5 11,2  | LAR
  5  3,7  - LEA
  5 12,3  | LED
  5 12,3  | LEU
  5 10,3  | LIAS
  5  2,14 - LID
  5 12,2  | LUE
  5  2,7  - LUE
  5  1,14 - LUI
  5  2,8  - LUO
  5  1,10 | LUT
  5  1,15 - LUT
  5  9,3  | PA
  5 12,3  | PE
  5  3,7  - PE
  5 10,3  | PI
  5  2,14 - PI
  5  3,8  - PO
  5 13,4  | RAD
  5 10,6  | SUD
  5  1,12 | THA
  5  1,14 - UDI
  5  2,8  - UDO
  5  9,2  | ULA
  5 12,2  | ULE
  5  2,7  - ULE
  5  1,10 | ULT
  5  1,15 - ULT
  5 13,3  | URA
  5  1,11 | UTU
  4  2,8  - ADO
  4  2,14 - AID
  4  2,14 - AIL
  4  9,2  | ALA
  4 12,2  | ALE
  4  2,7  - ALE
  4  1,10 | ALT
  4  1,15 - ALT
  4  2,8  - DAO
  4  1,10 | DAT
  4  1,15 - DAT
  4 12,3  | DEA
  4  2,14 - DIA
  4 12,4  | EAU
  4 12,4  | EH
  4  9,3  | HA
  4 12,3  | HE
  4  3,7  - HE
  4 10,3  | HI
  4  2,14 - HI
  4  3,8  - HO
  4 13,3  | HR
  4 10,4  | IDS
  4  3,14 - ILA
  4  1,14 - LAI
  4  2,8  - LAO
  4  1,10 | LAT
  4  1,15 - LAT
  4 12,3  | LEA
  4  6,3  | LU
  4 10,6  | SAD
  4  1,12 | TAD
  4  1,12 | TAL
  4  1,12 | TAU
  4  1,12 | TH
  4  1,12 | TUA
  4  7,3  | UN
  4  1,11 | UTA
  3  7,3  | AN
  3 11,3  | AR
  3  9,3  | DA
  3 10,8  | DA
  3 12,3  | DE
  3  3,7  - DE
  3 10,3  | DI
  3  2,14 - DI
  3  3,8  - DO
  3 12,4  | ED
  3 12,4  | EL
  3  3,14 - ID
  3  9,3  | LA
  3 10,3  | LI
  3  2,14 - LI
  3  3,8  - LO
  3  1,11 | UT
  3  2,15 - UT
  2  9,3  | AA
  2 12,3  | AE
  2  3,7  - AE
  2 10,3  | AI
  2  2,14 - AI
  2 13,3  | AR
  2  1,11 | AT
  2  2,15 - AT
  2  1,12 | TA
