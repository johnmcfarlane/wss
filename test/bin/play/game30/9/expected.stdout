 42 12,1  | UPHELD
 33 12,1  | PULED
 30 12,1  | PALED
 27 12,1  | DUPE
 27 12,1  | HALED
 27  1,15 - PATH
 27 12,1  | PULE
 24 10,1  | APHIDS
 24 12,1  | PALE
 24  3,15 - THUD
 22 10,1  | PLAIDS
 21 12,1  | HADE
 21 12,1  | HALE
 21  1,15 - LATH
 21 12,1  | LUDE
 21 10,3  - PAH
 21  1,7  - PULED
 21  3,15 - TUP
 20  2,11 | DHAL
 20 10,1  | HALIDS
 20  1,7  - PALED
 20  2,14 - PILAU
 19 10,3  - PAD
 19 13,2  | PURDAH
 18 12,1  | DALE
 18 10,1  | DULIAS
 18 12,3  - HAP
 18 12,1  | LADE
 18  1,15 - LATU
 18 10,2  | PAILS
 18  1,9  | PHUT
 18 10,1  | PULI
 18  3,15 - TAP
 17  3,5  | DAHL
 17 10,3  - HAD
 17  1,7  - HALED
 17  2,12 | HAP
 17  1,9  | PHAT
 17  7,1  | PLAN
 17  9,1  | PULA
 17  1,7  - PULE
 17 10,6  | SADDHU
 16 12,3  - DAHL
 16 12,3  - DAP
 16 10,8  | DUP
 16 10,2  | HAILS
 16 13,2  | HARP
 16 12,3  - HAUL
 16 10,1  | PADI
 16  3,5  | PAH
 16  1,7  - PALE
 16 10,2  | PHI
 16  1,9  | PLAT
 16 13,2  | PURDA
 16  5,12 | WHUP
 15 10,8  | DAP
 15  3,5  | DHAL
 15  2,12 | HAH
 15 10,3  - LAD
 15  3,5  | PUD
 15 13,2  | PURL
 15  5,12 | WHAP
 14 12,3  - DAH
 14 10,8  | DAHL
 14  3,5  | DUAL
 14 12,3  - HAD
 14  9,1  | HULA
 14  6,1  | LUAU
 14 10,3  - PA
 14  3,5  | PAD
 14  3,6  | PAH
 14 13,2  | PARD
 13 10,7  - ALP
 13 10,7  - AULD
 13  3,5  | DUH
 13 10,8  | DUH
 13  1,7  - DUPE
 13  1,7  - HADE
 13  1,7  - HALE
 13  1,9  | HALT
 13  1,9  | HAUT
 13 13,2  | HURL
 13  3,4  | PULA
 13 10,6  | SADHU
 12  1,8  | ADULT
 12 11,3  - AH
 12  3,4  | AULD
 12  3,5  | DAH
 12  3,6  | DAH
 12 10,8  | DAH
 12 12,3  - DAL
 12 10,3  - HA
 12  3,5  | HAD
 12 13,2  | HARD
 12 13,2  | HARL
 12  3,7  - HEP
 12  3,4  | HULA
 12  9,3  - PA
 12  2,14 | PA
 12  3,6  | PAD
 12  3,6  | PAL
 12 13,3  | PRAHU
 12 13,4  | RALPH
 12  3,15 - TAD
 12  3,15 - TAU
 12  6,2  | ULU
 12  7,3  | UNAU
 12  2,15 - UTA
 11 10,7  - AH
 11 12,2  | ALEPH
 11 13,3  - ALP
 11 13,2  | DURAL
 11  3,6  | HAD
 11  3,4  | LAUD
 11  1,7  - LUDE
 11  3,7  - PEH
 11  1,14 - PHIAL
 11  2,7  - PLED
 11 13,2  | PUR
 11 11,2  | PURL
 11  7,1  | ULAN
 11 13,3  | URP
 11  5,12 | WHA
 10 11,3  - AD
 10  9,3  - AH
 10  3,7  | AH
 10 10,2  | AHI
 10  2,7  - APED
 10  3,6  | DAL
 10  1,7  - DALE
 10  1,9  | DAUT
 10 10,8  | DHAL
 10 10,8  | DUD
 10 10,2  | DUI
 10  9,3  - HA
 10 12,3  - HA
 10  2,14 | HA
 10 11,2  | HARP
 10 12,3  | HELP
 10  2,7  - HUED
 10 11,2  | HURL
 10 10,3  - LA
 10  3,5  | LAD
 10  3,6  | LAD
 10  1,7  - LADE
 10 13,2  | LARD
 10  1,10 - LATI
 10 13,2  | PAR
 10 12,3  | PEDAL
 10 12,2  | PLEAD
 10  5,12 | WAP
 10  5,12 | WAUL
 10  5,12 | WUD
  9 10,7  - AD
  9 13,3  - AH
  9  2,7  - AHED
  9 10,7  - AL
  9 10,8  | DAD
  9 10,8  | DAL
  9 10,8  | DUAD
  9 10,8  | DUAL
  9  2,7  - DUEL
  9 13,2  | DURA
  9 12,4  | EPHA
  9  2,7  - HAED
  9 12,3  | HEAP
  9 10,3  | HIPS
  9 12,3  | LEHUA
  9 11,2  | PARD
  9  1,10 | PATH
  9  3,7  - PED
  9  2,7  - PLEA
  9 12,2  | PLED
  9 13,3  | PRAU
  9 11,2  | PUR
  9  3,6  | UH
  9  1,8  - UPDO
  9 11,3  | URP
  8  3,7  | AD
  8  3,6  | AH
  8  9,3  - AL
  8  3,7  | AL
  8 12,2  | APED
  8  2,13 - AR
  8 10,3  | DIPS
  8 11,2  | DURA
  8  2,12 | HA
  8 11,2  | HARD
  8 11,2  | HARL
  8 12,3  | HELD
  8 12,3  | HEP
  8  2,14 - HIP
  8 12,2  | HUED
  8  9,3  - LA
  8  2,14 | LA
  8 12,3  | LEAP
  8  2,14 - LIPA
  8 10,3  | LIPS
  8  1,14 - PAID
  8  1,14 - PAIL
  8 12,3  | PEAL
  8 12,3  | PEH
  8  1,14 - PHI
  8  1,10 | PHT
  8  1,15 - PHT
  8  2,14 - PIAL
  8 12,2  | PLEA
  8  7,2  | PUN
  8  1,12 | THUD
  8  5,12 | WAD
  7 13,3  - AD
  7 12,2  | AHED
  7 13,3  - AL
  7  3,7  - DEL
  7  2,14 - DIP
  7 12,2  | DUEL
  7  4,7  - EH
  7 12,2  | HAED
  7  1,14 - HAIL
  7  1,8  - HALO
  7 12,3  | HEAD
  7 12,3  | HEAL
  7  2,14 - HILA
  7  7,2  | HUN
  7 11,2  | LARD
  7  1,10 | LATH
  7  3,7  - LED
  7  3,7  - LEU
  7 12,3  | LEUD
  7  2,14 - LIP
  7  3,6  | PA
  7  7,2  | PAN
  7 11,2  | PAR
  7  3,7  - PEA
  7 12,3  | PED
  7 10,3  | PIAS
  7  2,14 - PIU
  7  1,10 | PUT
  7  1,15 - PUT
  7 13,4  | RAP
  7 10,6  | SUDD
  7  1,12 | TUP
  7  2,8  - UPO
  7 11,3  | URD
  7 13,3  | URD
  6  3,6  | AD
  6 12,2  | APE
  6  2,7  - APE
  6  2,8  - APO
  6  1,10 | APT
  6  1,15 - APT
  6 12,3  | DEAL
  6  2,14 - DIAL
  6  7,2  | DUN
  6 12,4  | EDH
  6  3,6  | HA
  6  2,14 - HID
  6 12,2  | HUE
  6  2,7  - HUE
  6  1,10 | HUT
  6  1,15 - HUT
  6  1,14 - LAID
  6 13,2  | LAR
  6  1,10 | LATU
  6 12,3  | LEAD
  6 10,3  | LIDS
  6  1,10 | PAT
  6  1,15 - PAT
  6 12,3  | PEA
  6  2,14 - PIA
  6 13,4  | RAH
  6  3,15 - TA
  6  1,12 | TAP
  5  9,2  | AHA
  5  1,14 - AHI
  5 10,3  | AIDS
  5 10,3  | AILS
  5  7,2  | DAN
  5 12,3  | DEL
  5 12,2  | DUE
  5  2,7  - DUE
  5  1,14 - DUI
  5  2,8  - DUO
  5  4,7  - ED
  5  4,7  - EL
  5 12,4  | ELD
  5 12,2  | HAE
  5  2,7  - HAE
  5  2,8  - HAO
  5  1,10 | HAT
  5  1,15 - HAT
  5  3,6  | LA
  5 11,2  | LAR
  5  3,7  - LEA
  5 12,3  | LED
  5 12,3  | LEU
  5  2,14 - LID
  5  9,3  | PA
  5 10,3  | PI
  5  2,14 - PI
  5 13,4  | RAD
  5  2,8  - UDO
  4  2,8  - ADO
  4  2,14 - AID
  4  2,14 - AIL
  4  9,2  | ALA
  4 12,2  | ALE
  4  2,7  - ALE
  4  1,10 | ALT
  4  1,15 - ALT
  4 12,4  | EAU
  4 12,4  | EH
  4  9,3  | HA
  4 12,3  | HE
  4  3,7  - HE
  4 10,3  | HI
  4  2,14 - HI
  4  3,8  - HO
  4 10,4  | IDS
  4  1,10 | LAT
  4  1,15 - LAT
  4 12,3  | LEA
  4 10,6  | SAD
  4  1,12 | TAD
  4  1,12 | TAU
  4  7,3  | UN
  4  1,11 | UTA
  3  7,3  | AN
  3 11,3  | AR
  3 12,3  | DE
  3  3,7  - DE
  3  3,8  - DO
  3 12,4  | ED
  3 12,4  | EL
  3  3,14 - ID
  3  9,3  | LA
  3 10,3  | LI
  3  2,14 - LI
  3  3,8  - LO
  3  1,11 | UT
  3  2,15 - UT
  2  9,3  | AA
  2 12,3  | AE
  2  3,7  - AE
  2 10,3  | AI
  2  2,14 - AI
  2 13,3  | AR
  2  1,11 | AT
  2  2,15 - AT
  2  1,12 | TA
